### PR TITLE
adding new button styles; adding ProjectThumbnail component

### PR DESCRIPTION
**Pull request by gvn**
I think we should rename `Tile` to something else. 

Maybe `ProjectThumbnail`?

### CODE REVIEW

File: www_src/pages/make/make.jsx [hunk 1]
```diff
@@ -24,14 +24,9 @@ var Make = React.createClass({
...
-          <div className="inner">
-            <div className="img">
-              <img src="../../img/toucan.svg" />
-            </div>
-          </div>
-        </Link>
+      <div id="make">
+        <button className="btn full teal">+ Create a Project</button>
+        <Tile/>
         <p><button onClick={this.onClick}>{this.state.test ? 'on' : 'off'}</button></p>
```
Comment (gvn) [inline www_src/pages/make/make.jsx]: What is this for?

Comment (k88hudson) [inline www_src/pages/make/make.jsx]: Ah, this was a test thing for testing state persistence in android. feel free to remove

File: www_src/components/button/button.less [hunk 2]
```diff
@@ -29,4 +30,13 @@
       margin-left: 0.3em;
     }
   }
+  &.full {
```
Comment (k88hudson) [inline www_src/components/button/button.less]: There is already a `btn-block` style for this, you should probably merge them

**Comment by gvn** [comment]
Ok, this is ready for another review pass @k88hudson

**Comment by k88hudson** [comment]
@gvn only issue left is the teal box-shadow is overriding the default :active state

**Pull request merged**